kratos/sources/model_part_io.cpp

```diff
@@ -4714,16 +4709,27 @@ namespace Kratos
         mpStream.swap(newStream);
     }
 
-    inline void ModelPartIO::CreatePartition(unsigned int number_of_threads,const int number_of_rows, DenseVector<unsigned int>& partitions)
+    inline void ModelPartIO::CreatePartition(unsigned int NumberOfThreads,const int number_of_rows, DenseVector<unsigned int>& partitions)
     {
-        partitions.resize(number_of_threads+1);
-        int partition_size = number_of_rows / number_of_threads;
+        partitions.resize(NumberOfThreads+1);
+        int partition_size = number_of_rows / NumberOfThreads;
         partitions[0] = 0;
-        partitions[number_of_threads] = number_of_rows;
-        for(unsigned int i = 1; i<number_of_threads; i++)
+        partitions[NumberOfThreads] = number_of_rows;
+        for(unsigned int i = 1; i<NumberOfThreads; i++)
             partitions[i] = partitions[i-1] + partition_size ;
     }
 
+    inline void ModelPartIO::EndSection(const std::string& rString)
+    {
```

Comment:
Why you have added this one? The SkipBlock deosn't do the same?

Reply:
I think I tried, but right I don't remember if I did, I will try later.

Reply:
You were right, this is duplicated, I removed it